README.md

```diff
@@ -0,0 +1,10 @@
+# Fish Invaders
+
+## Installation
+
+- Ensure that you have `pygame` installed
+
+
+## Run
+
+`python app.py`
```

Comment:
This README.md is just a text file so that other users can see how they can install + run Fish Invaders for themselves.